server/plugin/plugin.go

```diff
@@ -163,7 +164,7 @@ func (p *plugin) GetConfiguration() *config.Config {
 	return p.config
 }
 
-type plugin struct {
```

Comment:
1/5 it's OK to use `plugin` here. There is only one plugin in the code base so there is no room for confusion.

Reply:
It was a bit difficult to distinguish between `Plugin` the interface and `plugin` the struct, I don't know if it's a Go convention though.

Reply:
We don't